setup.py

```diff
@@ -43,5 +43,6 @@
         # tensorflow!=2.2.0,!=2.3.0 because of https://github.com/advisories/GHSA-8fxw-76px-3rxv
         "tensorflow>=2.1,!=2.2.0,!=2.3.0",
         "tensorflow-probability>=0.9",
+        "sklearn"
```

Comment:
huh, I'm a bit reluctant to add this dependency just for this standardization thingie :|